### ISSUE TITLE
Typescript 4.5.4 errors with Vue.install

### ISSUE THREAD

**Issue by drytoastman** [tserkov/vue-plugin-load-script#33]
I updated a project's packages and one of those was moving to a newer typescript (4.5.4).   Somewhere in there calling **Vue.use(LoadScript)** caued the following typescript check errors.

```
Overload 1 of 2, '(plugin: PluginObject<unknown> | PluginFunction<unknown>, options?: unknown): VueConstructor<Vue>', gave the following error.
    Argument of type 'typeof import("C:/Users/bwilson/sc/ts/node_modules/vue-plugin-load-script/index")' is not assignable to parameter of type 'PluginObject<unknown> | PluginFunction<unknown>'.     
      Type 'typeof import("XXX/node_modules/vue-plugin-load-script/index")' is not assignable to type 'PluginFunction<unknown>'.
        Type 'typeof import(XXX/node_modules/vue-plugin-load-script/index")' provides no match for the signature '(Vue: VueConstructor<Vue>, options?: unknown): void'.
  Overload 2 of 2, '(plugin: PluginObject<any> | PluginFunction<any>, ...options: any[]): VueConstructor<Vue>', gave the following error.
    Argument of type 'typeof import("XXX/node_modules/vue-plugin-load-script/index")' is not assignable to parameter of type 'PluginObject<any> | PluginFunction<any>'.
      Type 'typeof import("XXX/node_modules/vue-plugin-load-script/index")' is not assignable to type 'PluginFunction<any>'.
        Type 'typeof import("XXX/node_modules/vue-plugin-load-script/index")' provides no match for the signature '(Vue: VueConstructor<Vue>, options?: any): void'.
```

I was able to clear the error by adding the following to your index.d.ts:

```
export default class LoadScript {
  static install: PluginFunction<undefined>;
}
```

I'm not a typescript expert so I don't know if that is the correct thing to do.

**Comment by drytoastman** [comment]
Also added an import at the top: 
```
import { PluginFunction } from 'vue'
```

**Comment by tserkov** [comment]
Hi @drytoastman, if you were using the Vue 3 (vue3/v2.x.x) branch, I've just fixed this in v2.1.1, using your suggestions -- thanks! Please re-open if you were using the Vue 2 (master/v1.x.x) branch!
